### PR TITLE
Add missing semicolon in escaping of backslash

### PR DESCRIPTION
**Pull request by zuphilip**
This seems to fix the encoding problem mentioned in #60
without any other negative effect.

### CODE REVIEW

File: web/ocr-fileformat.js [hunk 1]
```diff
@@ -45,7 +45,6 @@ function escapeHTML(str) {
         replace(/</g, '&lt;').
         replace(/"/g, '&quot;').
         replace(/'/g, '&#39;').
-        replace(/\//g, '&#x2F').
```
Comment (zuphilip) [inline web/ocr-fileformat.js]: Maybe adding a semi-colon is enough here...

Comment (stweil) [reply web/ocr-fileformat.js]: Great, yes. https://digi.bib.uni-mannheim.de/ocr-fileformat-test/ now includes that fix.

**Comment by kba** [comment]
Why must strings use double quotes? Not that I'm against uniform coding style but I'd use eslint for that. Double / single quotes are interchangeable (apart from escaping quotes) in JS, unlike in PHP, Perl. These stylistic changes make it hard to spot what actually changed in this PR.

**Comment by zuphilip** [comment]
The explanation for consistent use of quotes: https://eslint.org/docs/rules/quotes . Do you think we should not try to follow that?

@kba Would it help, when the commit for codacy is taken out here and added in a separate PR?

**Comment by stweil** [comment]
Personally I prefer pull requests which address a single issue.

**Comment by zuphilip** [comment]
Okay, I splitted the commits into several PR. Can you look at them and hopefully merge them?

**Comment by stweil** [comment]
Thanks. https://digi.bib.uni-mannheim.de/ocr-fileformat-test/ now includes the latest fixes.

**Pull request merged**